app/scripts/metamask-controller.js

```diff
@@ -1167,6 +1177,10 @@ export default class MetamaskController extends EventEmitter {
           {
             snapId: process.env.PERMISSIONS_KERNEL_SNAP_ID,
             handleRequest: this.handleSnapRequest.bind(this),
+            onBeforeRequest: () =>
+              (rpcBlockingMiddlewareState.isBlocked = true),
+            onAfterRequest: () =>
+              (rpcBlockingMiddlewareState.isBlocked = false),
```

Comment:
```suggestion
            onBeforeRequest: () =>
              { rpcBlockingMiddlewareState.isBlocked = true },
            onAfterRequest: () =>
              { rpcBlockingMiddlewareState.isBlocked = false },
```

don't need to return the value, but doesn't really matter